### ISSUE TITLE
False Positive - react/no-multi-comp

### ISSUE THREAD

**Issue by rastalamm** [yannickcr/eslint-plugin-react#2612]
For the below example, unless I'm not understanding this rule properly, I'm seeing error. 

ESLint setting
`"react/no-multi-comp": ["error", { ignoreStateless: true }],`

```
class One extends PureComponent {
  render() {
    return <div>One</div>;
  }
}

class Two extends PureComponent {
  render() {
    return <div>Two</div>;
  }
}
```

I have two class based Components in a single file. Neither of them get/set a state. 

The documentation doesn't mention anything about multiple class definitions in a file. 

Running `v7.19.0`

**Comment by ljharb** [comment]
"ignoreStateless" means, ignore function components. All class components are stateful.

**Comment by rastalamm** [comment]
> All class components are stateful.

This is what I was missing! Thanks for the quick response 👍

**Comment by ljharb** [comment]
A PR to improve the docs to make that clear would be appreciated.
